### PR TITLE
red is beautiful

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Comment by you-leee** [comment]
I think we also need to delete/fix all the usages of DataTables.propertyTableName

**Comment by katherinebernstein** [comment]
yeah, I left it in now for mechanic though

**Comment by you-leee** [comment]
It's actually used in a lot of places like EdmService.getPropertyUsageSummary or PostgresEdmManager.updatePropertyTypeFqn

**Comment by katherinebernstein** [comment]
oh yeah those are all broken on staging right now

File: src/main/java/com/openlattice/edm/PostgresEdmManager.java [hunk 1]
```diff
@@ -173,20 +119,29 @@ public void createEntitySet( EntitySet entitySet, Collection<PropertyType> prope
...
-                        ENTITY_SET_ID_FIELD,
-                        propertyTableName,
-                        ENTITY_SETS,
-                        NAME_FIELD );
+    public Iterable<PropertyUsageSummary> getPropertyUsageSummary( UUID propertyTypeId ) {
+        final var wrappedEntitySetsTableName = "wrapped_entity_sets";
+        final var getPropertyTypeSummary =
+                String.format( "WITH %1$s AS (SELECT %2$s, %3$s AS %4$s, %5$s FROM %6$s) " +
+                                "SELECT %2$s, %4$s, %7$s, COUNT(*) FROM %8$s LEFT JOIN %1$s ON %7$s = %1$s.id " +
+                                "WHERE %9$s > 0 AND %10$s = ? GROUP BY ( %2$s , %4$s, %7$s )",
```
Comment (katherinebernstein) [inline src/main/java/com/openlattice/edm/PostgresEdmManager.java]: this hurts my brain just a little bit

Comment (you-leee) [reply src/main/java/com/openlattice/edm/PostgresEdmManager.java]: Yeah, it was really fun to check the order :D

**Pull request merged**